Xamarin.Forms.Core/AutomationProperties.cs

```diff
@@ -1,19 +1,19 @@
 ﻿﻿
 namespace Xamarin.Forms
 {
-	public class Accessibility
+	public class AutomationProperties
```

Comment:
@StephaneDelcroix 2.3.5-pre1 is when this class first appeared in a NuGet release.